procedures/unit-testing-tracing.ipf

```diff
@@ -280,26 +284,33 @@ static Function WriteProcList(string procWinList)
 
 	numProcs = ItemsInList(procWinList)
 
-	newCode = "Make/FREE/T wt = {\\\r"
-	for(i = 0; i < numProcs - 1; i += 1)
-		newCode += "\"" + StringFromList(i, procWinList) + "\",\\\r"
-	endfor
-	newCode += "\"" + StringFromList(numProcs - 1, procWinList) + "\"\\\r"
-	newCode += "}\r"
-	newCode += "return wt\r"
-	WAVE/T wNewCode = ListToTextWave(newCode, "\r")
+	if(numProcs)
+		newCode = "Make/FREE/T wt = {\\\n"
```

Comment:
What is the change of all line endings fixing here?

Reply:
The original idea was that this function produce the same empty file as it is currently checked in. The current line ending prevented this so I have fixed it. After reordering the commit this change does no longer exists.